### PR TITLE
Display an Intermediary "Approve"Button Before "Swap" Button

### PR DESCRIPTION
**Pull request by davidkoller318**
Hi, @chillvybz 
I added logic for issue https://github.com/polkaswitch/polkaswitch-ui/issues/14

Please review it. 
Thank you.

### CODE REVIEW

File: src/client/js/components/partials/swap/SwapWidget.jsx [hunk 1]
```diff
@@ -237,7 +243,7 @@ export default class SwapOrderWidget extends Component {
             from={this.state.from}
             fromAmount={this.state.fromAmount}
             toAmount={this.state.toAmount}
-            availableBalance={this.state.availableBalance}
```
Comment (chillvybz) [inline src/client/js/components/partials/swap/SwapWidget.jsx:240]: why was availableBalance removed here?

**Comment by chillvybz** [comment]
Otherwise this looks good. Please also fix conflicts.

File: src/client/js/components/partials/swap/SwapWidget.jsx (hunk 1 above)
Comment (davidkoller318) [reply src/client/js/components/partials/swap/SwapWidget.jsx:240]: Sorry, I will check it.

**Comment by davidkoller318** [comment]
> Otherwise this looks good. Please also fix conflicts.

Hi, @chillvybz . I fixed conflict and fixed it. 
you can merge it now. 

Thank you

**Pull request merged**